### DISCUSSION TITLE
GLPI Agent 1.11 - Deploy Error

### DISCUSSION THREAD

**Discussion by CSpencerLTD** [glpi-project/glpi-agent#786] [Q&A]
### Bug reporting acknowledgment

Yes, I read it

### Professional support

None

### Describe the bug

We've found a strange issue with the 'Deploy' task on GLPI Agent.
We've recently fixed the taskscheduler automatic action on our GLPI server and now deployment tasks are automatically being prepared on PC's, which is expected.

However, the tasks that are sent/prepared for the PC's go straight into Error (all at the same time), whereas if we deploy a task manually (force start), it works without any issues.

### To reproduce

1. Install GLPI Agent onto a PC
2. Force Inventory on the GLPI Agent
3. Check the 'Tasks/Groups' on the PC in GLPI and check if the tasks are 'prepared'... 
4. Force Inventory again, wait 1-2 minutes and refresh the GLPI Tasks/Groups page... now the tasks have gone into Error.

### Expected behavior

We expect GLPI Task Deploy to work automatically if the PC falls into one of the inventory groups which has software deployed to it.

### Operating system

Windows

### GLPI Agent version

v1.11, v1.10, v1.9, v1.8, v1.7.3

### GLPI version

10.0.15

### GLPIInventory plugin or other plugin version

GLPI Inventory v1.3.5

**Comment by g-bougard** [comment]
Hi @CSpencerLTD 

for such kind of support, you really deserve a professional support and should consider to subscribe: https://services.glpi-network.com/#offers

Such comportment is generally triggered by a misuse of the solution. So you must review in details all your configurations (including glpi-agent one) and also check what happens on glpi-agent by enabling debug=2 before restarting the service.

**Comment by CSpencerLTD** [comment]
@g-bougard 

I don't understand what's wrong though? If we manually select a software deployment task and press Force Start, it deploys without any issues.

Is this a known issue for GLPI Server 10.0.0.15 ?

**Reply by g-bougard** [reply]
You need to update glpi and also, and maybe first, glpi-inventory-plugin which is the plugin managing deploy tasks.

Actually, I don't see any evidence of glpi-agent issue in the discussion.

**Reply by CSpencerLTD** [reply]
We will update GLPI and the plugin tomorrow morning, i'll let you know if the issue still happens :) Thanks for your help @g-bougard
